finished header for mobile tablet desktop

robertSovar opened the pull request:
finished header for all queries, and fixing the modal

robertSovar committed:
finished header for mobile tablet desktop

finished header for all queries, and fixing the modal

NituAlexandru commented on src/partials/header.html:
```diff
@@ -1,37 +1,28 @@
       </svg>
     </a>
   </div>
+  <ul class="header-navigation-bar">
```
Imbraca lista in tagul nav - ca sa folosim semnatica cum trebuie

NituAlexandru commented on src/partials/header.html:
```diff
@@ -41,11 +32,53 @@
+  <!-- Header Modal -->
+
+  <div class="header-modal header-modal-is-hidden" data-modal-header>
+    <div class="header-modal-content">
```
Foloseste nav in loc de div - semantica

robertSovar committed:
updating header

replacing the semantic changes div into nav, adding animations for all the media queries

robertSovar committed:
update header

deleting something that i wanted to do but forgot to delete html code.
